src/entities/user.ts

```diff
@@ -29,7 +32,38 @@ export class User {
         return this.phone;
     }
 
+    getRole(): Roles {
+        return this.role;
+    }
+
     public static create(data: UserData) {
         return new User(data);
     }
+
+    public verifyPassword(password: string): boolean {
+        //temporary solution
+        if (password !== this.password) {
+            return false;
+        } else {
+            return true;
+        }
+    }
+
+    public updateUser({newLogin, newPassword, newRole, newPhone}) {
+        if (newLogin) {
+            this.login = newLogin;
+        }
+
+        if (newPassword) {
+            this.password = newPassword;
+        }
+
+        if (newRole) {
+            this.role = newRole;
+        }
+
+        if (newPhone) {
+            this.phone = newPhone;
+        }
+    }
```

Comment:
Updating entity in such way is not efficient.
Instead of this code I suggest following approach:
- create new entity (`Entity.create()`) using existing data & new data